docs/concepts/writing-config.md

```diff
@@ -0,0 +1,433 @@
+---
+title: Writing Istio Configuration
+headline: Writing Istio Configuration
+sidenav: doc-side-concepts-nav.html
+bodyclass: docs
+layout: docs
+type: markdown
+---
+
+{% capture overview %}
+This document describes how to write configuration that conforms to Istio's schemas. All configuration schemas in Istio are defined as protobuf messages. When in doubt, search for the protos.
+{% endcapture %}
+
+{% capture body %}
+## Translating to JSON
+The protobuf language guide defines [the canonical mapping between protobuf and JSON](https://developers.google.com/protocol-buffers/docs/proto3#json), refer to it as the source of truth. We provide a few specific examples that appear in the Mixer's configuration.
+
+### Proto `map` and `message` translate to JSON objects:
```

Comment:
Zack, you can fork the repo, then enable your personal github page (zackbutcher.github.io) via your GH settings. Then you enable GH pages in your fork of istio.github.io. Once you do these two, you can view the page via zackbutcher.github.io/istio.github.io/ 
(We should probably document this step. Not sure if we already did)